scripts/ci/provision.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+set -e
+
+echo '[Credentials]
+aws_access_key_id = not-a-real-id
+aws_secret_access_key = not-a-real-key' | tee ~/.boto
```

Comment:
Why are you using `tee` here? It seems like this can be handled with just a redirect.